### ISSUE TITLE
Impossible to use `hermione` with yarn pnp mode

### ISSUE THREAD

**Issue by epszaw** [gemini-testing/testplane#810]
When I try to use hermione in project which uses yarn pnp I receive this error all the time:

```
[12:56:16 GMT+0] Error: plugins-loader tried to access hermione-allure-hermione, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

Required package: hermione-allure-hermione
Required by: plugins-loader@npm:1.2.0 (via /home/runner/.yarn/berry/cache/plugins-loader-npm-1.2.0-c325a29b65-10c0.zip/node_modules/plugins-loader/lib/)
```

It happens due hermione try to resolve the plugin dynamically and yarn treats `hermione-`-prefixed path as dependency, which should be present in `package.json` file. Unfortunately, there isn't workaround to bypass the problem.

To reproduce the problem:

1. Create an empty directory
2. Initialize yarn in the directory `yarn init -2`
3. Bootstrap hermione project in the directory
4. Run `yarn install`
5. Run your test script

**Comment by DudaGod** [comment]
Hello.

Can you provide minimal reproducible example?

**Comment by epszaw** [comment]
Sure, will prepare the example today

**Comment by epszaw** [comment]
https://github.com/epszaw/hermione-yarn-pnp

**Comment by epszaw** [comment]
Changing node linker in context of yarn pnp doesn't look like a complete solution, user should be able to use any linker

**Comment by DudaGod** [comment]
I agree this is not a complete solution, so I did not close the issue and offered a temporary solution. We need to fix it.

**Comment by DudaGod** [comment]
Fixed in hermione@8.9.2 (and testplane@8.11.2)
